pkg/p2p/p2p.go

```diff
@@ -28,6 +28,13 @@ type Service interface {
 	Addresses() ([]ma.Multiaddr, error)
 }
 
+// DebugService extends the Service with method used for debugging.
+type DebugService interface {
```

Comment:
@acud, @pradovic Is this new superset interface an acceptable solution for the scope of this PR?